### ISSUE TITLE
Docker Desktop Warning Message to uninstall BlueStacks on MacOS Catalina 10.15.6

### ISSUE THREAD

**Issue by ajeetraina** [docker/for-mac#4863]
I have MacOS Catalina 10.15.6 running on my Mac Pro with the below configuration:

Docker Desktop Version:  2.3.5.0 
Release Type: Edge
Docker Engine: 19.03.13-beta2
BlueStack Version: 4.210.0

  - [ ] I have tried with the latest version of my channel (Stable or Edge)
Yes. I am running the latest 2.3.5.0 Edge Release
  - [ ] I have uploaded Diagnostics
Not Required.
  - Diagnostics ID:
Not Required

### Expected behavior
Bluestack and Docker Desktop should work together

### Actual behavior
A Notification from MacOS stating "Bluestacks will crash the operating system if Docker runs in the same MacOS boot sequence after Bluestacks has run. We recommend uninstalling Bluestacks in order to run Docker Desktop". If I select "Start Anyway" , Docker Desktop just operates fine. Its just said this error notifications gets displayed every time I switch on my MacBook.

### Information

  - macOS Version:

### Diagnostic logs

```
Docker for Mac: version...
ajeetraina@Ajeets-MacBook-Pro ~ % docker version         
Client: Docker Engine - Community
 Azure integration  0.1.12
 Version:           19.03.13-beta2
 API version:       1.40
 Go version:        go1.13.14
 Git commit:        ff3fbc9d55
 Built:             Mon Aug  3 14:58:48 2020
 OS/Arch:           darwin/amd64
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          19.03.13-beta2
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.14
  Git commit:       ff3fbc9d55
  Built:            Mon Aug  3 15:06:50 2020
  OS/Arch:          linux/amd64
  Experimental:     true
 containerd:
  Version:          v1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683

```

### Steps to reproduce the behavior

  1. Install Docker Desktop
  2. Install Bluestacks
 3. You will get the notification
 4. Whenever you switch off and on the system, this warning gets displayed.

**Comment by StefanScherer** [comment]
Hello my Captain :-)
That warning was added back in the early days due to issues mentioned in https://forums.docker.com/t/docker-cause-mac-os-crash-when-using-with-bluestack/19217/2
But there are newer comments that users don't see a problem. Maybe we should revisit it and drop the warning. 🤔

**Comment by ajeetraina** [comment]
@StefanScherer This warning message gets popped up every time Docker Desktop comes up after I power on the system. I am looking out for a way to get rid of this warning without uninstalling BlueStack.

**Comment by StefanScherer** [comment]
@ajeetraina There's no setting to disable the warning. Docker Desktop checks if `/Applications/BlueStacks.app` exists and pops up the warning. I don't know what happens if you rename the app folder, that would be the only workaround I see at the moment.
I'll install BlueStacks on my Mac and will do some tests.

**Comment by StefanScherer** [comment]
Oh you cannot actually rename it, it will install itself to `/Applications/BlueStacks.app` again. 
But I can't see any problems running Docker Desktop Edge 2.3.5.0 and BlueStacks 4.210.0 on macOS Catalina 10.15.6 at the same time.

**Comment by noma4i** [comment]
Can we get rid of that warning as BlueStacks is not a problem anymore? @StefanScherer

**Comment by stephen-turner** [comment]
Yes, we shall remove it.

**Comment by stephen-turner** [comment]
This is fixed in 2.5.0.0.

**Comment by ajeetraina** [comment]
Tested it with 2.5.0.0 and the issue is fixed. Thanks @StefanScherer @stephen-turner
